### PR TITLE
Preprocessors for RPi Pico

### PR DESCRIPTION
**Pull request by mthorley**
Updates to support OXRS_API for RPi Pico devices. 
Please refer to https://github.com/mthorley/pico-aqs for further details.

### CODE REVIEW

File: library.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name=OXRS-IO-API-ESP32-LIB
-version=2.6.0
+version=2.7.0
 author=OXRS Core Team
 maintainer=Ben Jones <ben.jones@gmail.com>
-sentence=ESP32 API library for Open eXtensible Rack System firmware
+sentence=ESP32 and PicoW API library for Open eXtensible Rack System firmware
```
Comment (sumnerboy12) [inline library.properties]: I wonder if we just change this to `API library for Open...`?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-# ESP32 API library for [OXRS](https://oxrs.io)
+# ESP32 and PicoW API library for [OXRS](https://oxrs.io)
```
Comment (sumnerboy12) [inline README.md]: Maybe change this title to `API library for...` and then underneath list the devices it supports?

Then it is easy to add to this list in the README.

Ideally we would rename the repo as well, but I think that can wait.

**Review by sumnerboy12 — CHANGES_REQUESTED** [review]
Looks really good Matt, just a few tweaks in the README etc but code-wise I think it looks fine. Thanks very much for contributing back!

**Comment by mthorley** [comment]
Updated to reflect better documentation as suggested.

**Review by moinmoin-sh — APPROVED** [review]
Looks good to me

**Review by sumnerboy12 — APPROVED** [review]

**Pull request merged**